src/com/facebook/buck/features/apple/projectV2/XcodeNativeTargetProjectWriter.java

```diff
@@ -195,6 +195,15 @@ public Result writeTargetToProject(
                       targetAttributes.lldbRemapScriptPath(),
                       nativeTarget);
 
+                  targetAttributes.customScriptPath().ifPresent( customScriptPath -> {
```

Comment:
It's optional to allow to run setting this config as empty, like `--config apple.xcode_custom_script=''`